core/modules/server/routes/get-tiddlers-json.js

```diff
@@ -13,13 +13,15 @@ GET /recipes/default/tiddlers/tiddlers.json?filter=<filter>
 "use strict";
 
 var DEFAULT_FILTER = "[all[tiddlers]!is[system]sort[title]]";
+var DEFAULT_INCLUDE_NON_TIDDLERS = false;
```

Comment:
Is there any reason for pulling this out as a constant? We tend to do this for readability (e.g. making the default tiddler readable and obvious) but here there's not much benefit.

Reply:
Especially together with your comment below you're right the apparent flexibility that comes from representing it as a default fallback parameter could cause more harm then good.
My intention was to improve readability and be consistent with the DEFAULT_FILTER, but I see they are actually semantically different from one another.